esmvaltool/cmorizers/data/formatters/datasets/agcd.py

```diff
@@ -0,0 +1,104 @@
+"""ESMValTool CMORizer for AGCD data.
+
+Tier
+    Tier 3: restricted dataset.
+
+Source
+    https://dx.doi.org/10.25914/hjqj-0x55
+
+Last access
+    20231121
+
+Download and processing instructions
+    Data from NCI project requiring an NCI account and access to Gadi
+    Processing is done on GADI
+
+"""
+import logging
+import os
+import re
+
+import iris
+
+from esmvaltool.cmorizers.data import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _get_filepaths(in_dir, basename):
+    """Find correct name of file (extend basename with timestamp)."""
+    regex = re.compile(basename)
+    return_files = []
+    for root, _, files in os.walk(in_dir, followlinks=True):
+
+        for filename in files:
+            if regex.match(filename):
+                return_files.append(os.path.join(root, filename))
+
+    return return_files
+
+
+def fix_data_var(cube, var):
+    """Convert units in cube for the variable."""
+    if var == 'pr':
+        cube = cube / (30 * 86400)  # ~ 30 days in month
+        cube.units = 'kg m-2 s-1'
+    elif var.startswith('tas'):  # other variables in v1
```

Comment:
probably safer if you used eg `elif var in ["tas", "tasmin", "tasmax"]:` - this bulletproofs the code for eg selecting a future variable called `tasmania` :grin: Plus, it is obvious to the reader what variables need be converted